engines/stark/services/userinterface.cpp

```diff
@@ -396,4 +399,31 @@ void UserInterface::performToggleSubtitle() {
 	_toggleSubtitle = false;
 }
 
+void UserInterface::cycleInventory(int step) {
+	Resources::KnowledgeSet *inventory = StarkGlobal->getInventory();
+	Common::Array<Resources::Item *> inventoryItems = inventory->listChildren<Resources::Item>(Resources::Item::kItemInventory);
+
+	int16 curItem = getSelectedInventoryItem();
+	int16 nextItem = curItem + step;
+
+	while (true) {
+		// Boundary cases
+		if (nextItem < -1) {
+			nextItem = inventoryItems.size() - 1;
+		}
+		if (nextItem > 0 && uint(nextItem) >= inventoryItems.size()) {
+			nextItem = -1;
+		}
+
+		// Item None, Hand, Eye, Mouth are skipped
+		if (nextItem < 0 || (inventoryItems[nextItem]->isEnabled() && nextItem > 3)) {
```

Comment:
Can I safely assume that the item None, Hand and Mouth are the first four items in the inventory?